utils/convertHexObjToVars.ts

```diff
@@ -0,0 +1,46 @@
+/**
```

Comment:
This file meant to be included? Hard-coded colours, console logs, just for local testing?

Reply:
It is but it isn't exported. (It isn't delivered to the users)
It's a utility for converting hex values to tailwindcss vars. Useful when starting a project. I plan to expand on this later by generating a color palette from a single color)